resources/js/Pages/SamplePage.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
Same as the comment above, I think the example page we include can already be something that we can use as a scaffolding for our app. Maybe we should rename this to be the homepage, and add some "coming soon" copy?

Reply:
yes, sure.